docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -167,4 +167,6 @@ Continue to iterate by adding more tests, more theories, and more code in the ma
 
 ### Additional resources
 
+-   **xUnit.net**. Official site. <br/>
```

Comment:
We haven't been displaying both descriptive text and the URL in the documentation except in the architecture ebooks. this should be \[**xUnit.net** official site](https://xunit.github.io)

Reply:
```suggestion
- [xUnit.net official site](https://xunit.github.io)
```

Reply:
Thanks for the tip @rpetrusha. 

I will make the necessary changes.